pypeit/utils.py

```diff
@@ -10,13 +10,12 @@
 import pickle
 import warnings
 import itertools
-from collections import deque
-from bisect import insort, bisect_left
 
 from IPython import embed
 
 import numpy as np
 from numpy.lib.stride_tricks import as_strided
+import bottleneck as bn
```

Comment:
The use of `as` is a pet-peeve of mine.  If this is an "industry standard", then I'll get over it.  Otherwise, I'd prefer we just do `import bottleneck` or `from bottleneck import move_median`.

Reply:
I agree @kbwestfall, except for standard abbreviations like np for numpy and plt for pyplot.

Reply:
bottleneck's github README uses `as`, which is why I used `as`, but Astropy doesn't so I'm happy to change it to just `import bottleneck`